docs/dcp2_system_design.rst

```diff
@@ -960,221 +960,91 @@ Project-level matrices
 
 Contributor-generated matrices (CGMs)
 -------------------------------------
+Managed Access CGMs
```

Comment:
We want to use this mechanism for CGMs of all projects, public and access controlled, so this is misleading. Please remove all mentions of access controls, managed access, data use etc.